### ISSUE TITLE
Install OpenHIM

### ISSUE THREAD

**Issue by maurya** [OpenConceptLab/DATIM-Metadata-Project#42]
Install OpenHIM.

Requirements:
Ubuntu 14.04
I've updated https://issues.openmrs.org/browse/ITSM-4023 to include instructions (I've adapted the instructions from https://github.com/jembi/openhim-core-js#getting-started-with-the-openhim-core)

**Comment by maurya** [comment]
Hi @bmamlin, I've updated the issues to include bash instructions in the ITSM ticket, it could be easily scripted in ansible or written as a bash script that could be run. Let me know how I can be of assistance.

**Comment by bmamlin** [comment]
I've updated [ITSM-4023](https://issues.openmrs.org/browse/ITSM-4023). We are currently blocked on (1) avoiding a certificate prompt during openhim-core-js install and (2) getting OpenHIM running in the Ubuntu 14.04 container.

**Comment by bmamlin** [comment]
We've resolved blockers on [ITSM-4023](https://issues.openmrs.org/browse/ITSM-4023). I will work on generating a PR to get this into Jetstream and report progress on the ITSM ticket & here.

**Comment by bmamlin** [comment]
I submitted a [pull request](https://github.com/openmrs/openmrs-contrib-ansible-docker-compose/pull/13) to add OpenHIM to the OpenMRS infrastructure.

**Comment by bmamlin** [comment]
So, we're very close to getting this done. The last remaining question is which ports need to be exposed externally. I've asked @maurya to answer this question  on [ITSM-4023](https://issues.openmrs.org/browse/ITSM-4023).

**Comment by bmamlin** [comment]
OpenHIM branch has been merged into ansible-docker-compose repo in OpenMRS infrastructure. All that's left is for Cintia to deploy it.

**Comment by bmamlin** [comment]
Per [ITSM-4023](https://issues.openmrs.org/browse/ITSM-4023), Cintia has gotten OpenHIM hosted on Jetstream at [https://openhim.openmrs.org/](https://openhim.openmrs.org/). For technical reasons, the API and core ports are on separate subdomains (openhim-api & openhim-core).

**Comment by paynejd** [comment]
Burke providing ssh access to Maurya
